### ISSUE TITLE
Dedicated applications database repository

### ISSUE THREAD

**Issue by lra** [lra/mackup#506]
Let's move the INI files in a dedicated repos that other projects can use, and other people can maintain.

Multiple talks going on:
- https://github.com/caskroom/homebrew-cask/issues/1135
- https://github.com/RichiH/vcsh/issues/66

As more projects with a need for those files get created, we need to make it available and open to the other to prevent any fork, which will just divide the community effort.

@vitorgalvao would you be up to be a maintainer of such a repo?

**Comment by josephholsten** [comment]
Anything I can do to help?

**Comment by vitorgalvao** [comment]
Yes, if the option for app bundles is added, [as discussed](https://twitter.com/vhgalvao/status/511467067611287553). Something like [`atom`](https://github.com/lra/mackup/blob/902e2eda610403e393a24960efee57ac41d202c6/mackup/applications/atom.cfg) would end up like

```ini
[application]
name = Atom
id = com.github.atom

[configuration_files]
Library/Preferences/com.github.atom.plist
.atom/config.cson
.atom/init.coffee
.atom/keymap.cson
.atom/keymaps
.atom/snippets.cson
.atom/styles.less
```

**Comment by lra** [comment]
The problem of out of date syncing is still relevant though, we would have to copy the files instead of linking, and the users would have to regularly run a sync command.

This is another discussion IMO =)

**Comment by vitorgalvao** [comment]
To be clear, I’m talking exclusively about adding that line to the INI files and not at all about adding the feature to mackup.

I very much understand your desire to keep mackup sync-only, but if I or anyone else wants to create a tool that is centred on copying, having the bundle identifier to overcome that limitation in mackup is useful.

My argument here is not that mackup must support bundle identifiers, but that this central repo we’re talking about, to be useful for other projects, should include it.

**Comment by lra** [comment]
Depends on https://github.com/lra/mackup/issues/610 anyway
